src/librustc_mir/shim.rs

```diff
@@ -213,10 +221,10 @@ fn build_drop_shim<'a, 'tcx>(tcx: ty::TyCtxt<'a, 'tcx, 'tcx>,
 }
 
 pub struct DropShimElaborator<'a, 'tcx: 'a> {
```

Comment:
Unlike the other use, it changes the "semantics" of the code. But nevermind.

concern resolved.